src/Accordion.php

```diff
@@ -166,7 +166,7 @@ public function renderView()
         }
 
         //Only set Accordion in Top container. Otherwise Nested accordion won't work.
-        if (!$this->owner instanceof AccordionSection && !$this->isDynamicSection()) {
+        if (!$this->getClosestOwner($this, '\atk4\ui\AccordionSection') && !$this->isDynamicSection()) {
```

Comment:
you can specify \atk4\ui\AccordionSection::class btw... not sure how it would work though. no need to fix!

Reply:
let's leave it as is for now because we don't use new `::class` syntax in other places too.